src/components/transactions/Repay/RepayModalContent.tsx

```diff
@@ -163,7 +162,7 @@ export const RepayModalContent = ({
         balance: maxBalance.toString(10),
       });
     }
-    setAssets(repayTokens);
+    setAssets(repayTokens.sort((a, b) => Number(a.balance) - Number(b.balance)));
```

Comment:
Would sorting by usd value here instead of balance be better UX?

Reply:
The only options in this modal are the underlying asset, the aToken, and a wrapped version of the assets which will all have the same USD price so sorting by balance will have the same affect

Reply:
Reverted this ordering change. Always showing the base asset first is the best UX. The issue of not being able to find these other options should be addressed separately in #732